lib/tasks/heroku.rake

```diff
@@ -1,7 +1,7 @@
 namespace :heroku do
   desc 'Heroku release task - runs on every code push, runs before postdeploy task'
   task release: :environment do
-    if ActiveRecord::SchemaMigration.table_exists?
+    if ActiveRecord::Base.connection.schema_migration.table_exists?
```

Comment:
This was mildly annoying to debug. Caused heroku release phase to fail.

Caused by "Remove deprecated ActiveRecord::Base config accessors"

[Context](https://github.com/rails/rails/blob/7-1-stable/activerecord/CHANGELOG.md#:~:text=Move%20ActiveRecord%3A%3AInternalMetadata%20to%20an%20independent%20object.)